src/test/java/io/rocketpartners/cloud/action/rest/TestRestGetActions.java

```diff
@@ -116,4 +119,108 @@ public void testPagination01() throws Exception
       assertEquals(830, total);
    }
 
+   @Test
+   public void testEw01() throws Exception
+   {
+      Service service = service();
+      Response res = null;
+      res = service.get(url("orders?ew(shipname,Chevalier)"));
+      ArrayNode data = res.data();
+      boolean assertion = true;
+      for (Object o : data)
+      {
+         ObjectNode js = (ObjectNode) o;
+         if (js.getString("shipname").endsWith("Chevalier"))
+            continue;
+         else
+         {
+            assertion = false;
+         }
+      }
+      assertTrue(data.size() > 0 && assertion);
+   }
+
+   @Test
+   public void testLt01() throws Exception
+   {
+      Service service = service();
+      Response res = null;
+      res = service.get(url("orders?limit=5&lt(freight,2)"));
+      boolean assertion = true;
+      ArrayNode data = res.data();
+      for (Object o : data)
+      {
+         ObjectNode js = (ObjectNode) o;
+         if (Float.parseFloat(js.getString("freight")) < 2)
+            continue;
+         else
+         {
+            assertion = false;
+         }
+      }
+      assertTrue(data.size() > 0 && assertion);
+   }
+
+   @Test
+   public void testLe01() throws Exception
+   {
+      Service service = service();
+      Response res = null;
+      boolean assertion = true;
+      res = service.get(url("orders?limit=5&le(freight,2)"));
+      ArrayNode data = res.data();
+      for (Object o : data)
+      {
+         ObjectNode js = (ObjectNode) o;
+         if (Float.parseFloat(js.getString("freight")) <= 2)
+            continue;
+         else
+         {
+            assertion = false;
+         }
+      }
+      assertTrue(data.size() > 0 && assertion);
+   }
+
+   @Test
+   public void testIn01() throws Exception
+   {
+      Service service = service();
+      Response res = null;
+      res = service.get(url("orders?in(orderid,10249,10258,10252)"));
+      boolean assertion = true;
+      ArrayNode data = res.data();
+
+      List<String> list = Arrays.asList("10249", "10258", "10252");
+      for (Object o : data)
+      {
+         ObjectNode js = (ObjectNode) o;
+         if (list.indexOf(js.getString("orderid")) > -1)
+            continue;
+         else
+         {
+            assertion = false;
+         }
+      }
+      assertEquals(3, data.length());
+      assertTrue(assertion);
+   }
+
+   @Test
+   public void testOut01() throws Exception
+   {
+      Service service = service();
+      Response res = null;
+      res = service.get(url("employees?out(employeeid,1,2)")).statusOk();
+      assertEquals(7, res.data().length());
```

Comment:
Please check the employeeid of the returned values to make sure the do not include 1 or 2.  Can leave the 7 length check but one of those seven could be 1 or 2.